### PR TITLE
get url form DOCKER_COMPOSE_DOWNLOAD_LINK if exist

### PR DESCRIPTION
**Pull request by MaxPeal**
## Description

get url form DOCKER_COMPOSE_DOWNLOAD_LINK if exist

## Why is this needed

to doit on the same way like the setup.sh

Fixes: #

## How Has This Been Tested?
just testit it localy

## How are existing users impacted? What migration steps/scripts do we need?

no migration steps needed

### CODE REVIEW

**Comment by gianarb** [comment]
Hello! Thank you @MaxPeal for your contribution! And welcome!!

A few things:

1. Do you mind explaining why you are introducing a new option in the `setup.sh`? You can write it `Why is this needed` the current section does not really explain to me why you need that variable.
2. Do you mind to squash your commits in a single one?
3. We use (shellcheck)[https://www.shellcheck.net/] to statically analyze bash scrits. Your changes are currently not complaint. Do you mind to fix them?

**Review by gianarb — CHANGES_REQUESTED** [review]
Hello! Thank you @MaxPeal for your contribution! And welcome!!

A few things:

1. Do you mind explaining why you are introducing a new option in the `setup.sh`? You can write it `Why is this needed` the current section does not really explain to me why you need that variable.
2. Do you mind to squash your commits in a single one?
3. We use (shellcheck)[https://www.shellcheck.net/] to statically analyze bash scrits. Your changes are currently not complaint. Do you mind to fix them?

File: deploy/vagrant/scripts/tinkerbell.sh [hunk 1]
```diff
@@ -16,12 +40,17 @@ setup_docker() (
 		gnupg-agent \
 		software-properties-common
 
-	curl -fsSL https://download.docker.com/linux/ubuntu/gpg |
+	local lsb_dist
```
Comment (tstromberg) [inline deploy/vagrant/scripts/tinkerbell.sh:43]: This work to support other LSB distributions should be moved to a separate PR.

**Comment by displague** [comment]
Fixes https://github.com/tinkerbell/tink/issues/518

**Comment by displague** [comment]
I've opened a new rebased, conflict merged, PR with some of the changes from this PR, @MaxPeal.

See https://github.com/tinkerbell/tink/pull/519

Any vagrant work will need to be moved to tinkerbell/sandbox.  I created an issue to track that #518, but closed it when I realized that vagrant was removed from this repo.

Closing this PR in favor of #519.

**Pull request closed without merging**